### PR TITLE
[MOSIP-29901] updated the README.md for storage classed

### PR DESCRIPTION
**Pull request by Rakshitha650**

### CODE REVIEW

**Review by syedsalman3753 — APPROVED** [review]
approved

File: mosip/aws/README.md [hunk 1]
```diff
@@ -59,8 +59,14 @@ kubectl apply -f sc.yaml
 ```
 * If the PV gets deleted (say cluster was retarted), then you will have to define a PV connecting to this instance of storage (you will need volume ID etc). TODO: how to do this?
 
-### LongHorn
-Install [LongHorn](../longhorn/README.md).
+## The following storage classes can be used:
```
Comment (ckm007) [inline mosip/aws/README.md:62]: this is aw specific section.. here we should have gp2 or gp3 or EBS or EFS mentioned.

File: mosip/on-prem/README.md [hunk 2]
```diff
@@ -161,8 +161,14 @@ kubectl apply -f https://rancher.e2e.mosip.net/v3/import/pdmkx6b4xxtpcd699gzwdtt
...
-## Longhorn
-* Install [Longhorn](../../longhorn/README.md) for persistent storage.
+## The following storage classes can be used:
+
+* [Vsphere storage class](https://github.com/vmware-archive/vsphere-storage-for-kubernetes): If you are already using VMware virtual machines, you can proceed with the vSphere storage class.
+* [NFS client provisioner storage class](../../nfs/README.md).
+* [ceph-csi](TODO Implementation in progress)
+* [Longhorn](../../longhorn/README.md)
+
+MOSIP is implementing `Longhorn` as its storage class solution for improved data management.
```
Comment (ckm007) [inline mosip/on-prem/README.md:171]: For time being considering Longhorn as a storage class.

**Comment by ckm007** [comment]
Did we add the same to reflect in our docs?

**Review by SwatiPeriwal — CHANGES_REQUESTED** [review]
please make the necessary changes and update

File: mosip/on-prem/README.md [hunk 3]
```diff
@@ -161,8 +161,14 @@ kubectl apply -f https://rancher.e2e.mosip.net/v3/import/pdmkx6b4xxtpcd699gzwdtt
 * Wait for few seconds after executing the command for the cluster to get verified.
 * Your cluster is now added to the rancher management server.
 
-## Longhorn
-* Install [Longhorn](../../longhorn/README.md) for persistent storage.
+## The following storage classes can be used:
```
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:164]: The heading cannot be a sentence.... You can have ## Storage Class and then write the sentence- The following storage classes can be used:

File: mosip/on-prem/README.md [hunk 4]
```diff
@@ -161,8 +161,14 @@ kubectl apply -f https://rancher.e2e.mosip.net/v3/import/pdmkx6b4xxtpcd699gzwdtt
...
+
+* [Vsphere storage class](https://github.com/vmware-archive/vsphere-storage-for-kubernetes): If you are already using VMware virtual machines, you can proceed with the vSphere storage class.
+* [NFS client provisioner storage class](../../nfs/README.md).
+* [ceph-csi](TODO Implementation in progress)
+* [Longhorn](../../longhorn/README.md)
+
+For time being considering Longhorn as a storage class.
```
Comment (SwatiPeriwal) [inline mosip/on-prem/README.md:171]: For time being considering Longhorn as a storage class. - This sentence is incorrect... For time being. let us consider Longhorn as a storage class. 9or pls check the technical context and update accordingly

**Review by SwatiPeriwal — CHANGES_REQUESTED** [review]
Pls update the doc

**Review by SwatiPeriwal — APPROVED** [review]
Please capitalize S in the heading ## Storage Classes

**Review by ckm007 — APPROVED** [review]

**Pull request merged**